source/blog/2021-11-09-announcing-hanami-200alpha3.html.markdown

```diff
@@ -0,0 +1,163 @@
+---
+title: Announcing Hanami v2.0.0.alpha3
+date: 2021-11-09 12:30:00 UTC
+tags: announcements
+author: Tim Riley
+image: true
+excerpt: >
+  Streamlined source directories, more flexible settings, better integrated actions, and the beginning of a monthly release cadence.
+---
+
+Hello Hanami community! It’s Tim here again, and I’m delighted to announce the release of Hanami 2.0.0.alpha3!
+
+This release represents months of work on several foundational aspects of the framework, and brings:
+
+- Streamlined source directories for easier-to-navigate project file trees
+- Reworked application settings, leveraging a regular Ruby class to give you full flexibility with your setting definitions
+- Reworked application routes, to match the above
+- Improvements to view rendering within actions
+- Framework settings ported to dry-configurable
```

Comment:
```suggestion
- Framework settings are now powered by `dry-configurable`
```

Reply:
Will adopt, but not the `code` style for dry-configurable, since that's now the dry-rb house style for naming our gems (they're just regular words, not code).